module/Entra/AdditionalFunctions/Get-EntraObjectSetting.ps1

```diff
@@ -0,0 +1,92 @@
+# ------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+# ------------------------------------------------------------------------------
+function Get-EntraObjectSetting {
```

Comment:
This cmldet should be similar to `Get-EntraObjectSetting`
E.g `Property` param is missing here

Reply:
fixed and added.